clients/destination.go

```diff
@@ -300,7 +301,20 @@ func (c *DestinationClient) Write2(ctx context.Context, tables schema.Tables, so
 	}
 	_, err = saveClient.CloseAndRecv()
 	if err != nil {
-		return fmt.Errorf("failed to CloseAndRecv client: %w", err)
+		st := status.Convert(err)
```

Comment:
We should probably put this into a function and replace https://github.com/cloudquery/plugin-sdk/pull/477 so we can get both the message and the details